app/views/host_mailer/pre_tea_time_nudge.markerb

```diff
@@ -1 +1,31 @@
-Template nudge email
+<% if @tea_time.attendees.present? %>?
+  Hey! Your tea time is on Thursday at 7:00pm. Now would be a good time to email your attendees to confirm they're in, offer a chance for them to cancel if needed, and share whatever other info you might need to. Try to do this today so they have sufficient notice.
```

Comment:
This shouldn't be Thursday at 7:00pm. It should be `<%= "#{tea_time.day}" %> at <%= "#{tea_time.start_time.strftime("%-l:%M%P" %>`